### ISSUE TITLE
[Assignment 3]  Code needs documentation

### ISSUE THREAD

**Issue by luisortizuriarte** [lorendoyle/clim680#6]
As you write your code, keep in mind to document what you are doing. This will help other people (like me!) understand what you are doing and how to offer help, and it will help future you understand what you wrote!

Please add documentation to your code using markdown cells and inline comments so that a reader can understand your code.

**Comment by pdirmeyer** [comment]
Agreed - I found it especially difficult to follow what you were doing, as you were clearly experimenting a lot with the data - many cells and no descriptions. Experimenting is fine - encouraged, even - but keep notes inline in your Notebook!

Specifically regarding your input data, what is in the file ts.nc? Temperature, obviously, and the range suggests it is from UFS hindcasts, but only one value per day - is this forecast data from different leads? I do not understand how you produced ts.nc - it needs to be described.

**Comment by lorendoyle** [comment]
I updated the notebooks to include documentation, and cleaned them up to only include the cells I needed. I have 3 separate notebooks (one for each variable). As for the data, from what I understand from what Qiu told me, is that she took 31 days for each month. UFS P7 is a retrospective run for the period from 2011 to 2018, and from that I chose 2011-2016. The runs are initialized twice per month (1st and 15th), and I only chose the 1st so as not to overload on space. The length of the forecast is 35 days with an output frequency of 6 hours, but when Qiu downloaded it she automatically resampled it to daily means, and took 31 days for each month.
